docker-compose.yml

```diff
@@ -92,6 +96,25 @@ services:
     deploy:
        restart_policy:
         condition: on-failure
+        
+        
+        
+  jobscheduler:    
+    container_name: jobscheduler
+    restart: on-failure
+    build:
+      context: .
+      dockerfile: JobScheduler/Dockerfile
+    depends_on:
+      - api
+      
+    stdin_open: true
+    tty: true
```

Comment:
I don't think these are necessary for service worker template, since it uses a hosted service. Have you tried without?